### PR TITLE
【1.1.10】Entry tenant label supports common configuration

### PR DESCRIPTION
**Pull request by huangKai-2323**
### What is the purpose of the change

Tenant tag support user-*

### Related issues/PRs

Related issues:#4391

### Brief change log

- Define the core abstraction and interfaces of the EngineConn Factory;
- Define the core abstraction and interfaces of Executor Manager.

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

File: linkis-public-enhancements/linkis-configuration/src/main/java/org/apache/linkis/configuration/restful/api/TenantConfigrationRestfulApi.java [hunk 1]
```diff
@@ -236,14 +236,15 @@ public Message checkUserCreator(
...
       }
-      if (creator.equals("*")) {
-        throw new ConfigurationException("Application Name can't be '*' ");
-      }
       String userName = ModuleUserUtils.getOperationUser(req, "checkUserCreator");
       if (!Configuration.isAdmin(userName)) {
         return Message.error("Failed to check-user-creator,msg: only administrators can configure");
       }
-      result = tenantConfigService.userExists(user, creator, tenantValue);
+      result = tenantConfigService.userExists(user, creator);
```
Comment (casionone) [inline linkis-public-enhancements/linkis-configuration/src/main/java/org/apache/linkis/configuration/restful/api/TenantConfigrationRestfulApi.java:243]: rename result  to isExist

File: linkis-public-enhancements/linkis-configuration/src/main/java/org/apache/linkis/configuration/service/impl/TenantConfigServiceImpl.java [hunk 2]
```diff
@@ -119,55 +118,56 @@ public void updateTenant(TenantVo tenantVo) throws ConfigurationException {
...
-      resultmap = HttpsUtil.sendHttp(null, null);
-      logger.info("ResourceMonitor  response  {}:", resultmap);
-    } catch (IOException e) {
-      logger.warn("failed to get ecmResource data");
+    if (!tenantVo.getCreator().equals("*")) {
+      AtomicReference<Boolean> tenantResult = new AtomicReference<>(false);
+      // Obtain the tenant information of the ECM list
+      Map<String, Object> resultmap = null;
+      try {
+        resultmap = HttpsUtil.sendHttp(null, null);
```
Comment (casionone) [inline linkis-public-enhancements/linkis-configuration/src/main/java/org/apache/linkis/configuration/service/impl/TenantConfigServiceImpl.java:132]: resource monitor？

File: linkis-public-enhancements/linkis-configuration/src/main/java/org/apache/linkis/configuration/restful/api/TenantConfigrationRestfulApi.java (hunk 1 above)
Comment (huangKai-2323) [reply linkis-public-enhancements/linkis-configuration/src/main/java/org/apache/linkis/configuration/restful/api/TenantConfigrationRestfulApi.java:243]: done

File: linkis-public-enhancements/linkis-configuration/src/main/java/org/apache/linkis/configuration/service/impl/TenantConfigServiceImpl.java (hunk 2 above)
Comment (huangKai-2323) [reply linkis-public-enhancements/linkis-configuration/src/main/java/org/apache/linkis/configuration/service/impl/TenantConfigServiceImpl.java:132]: done

**Pull request merged**